core/domain/takeout_service_test.py

```diff
@@ -671,7 +722,8 @@ def test_export_data_nontrivial(self):
                 'has_suggestion': self.THREAD_HAS_SUGGESTION,
                 'summary': self.THREAD_SUMMARY,
                 'message_count': self.THREAD_MESSAGE_COUNT,
-                'last_updated': feedback_thread_model.last_updated
+                'last_updated': utils.get_time_in_millisecs(
```

Comment:
Should change the key to last_updated_msec?

Ditto elsewhere...

Reply:
Done